src/status_im/contexts/settings/wallet/wallet_options/view.cljs

```diff
@@ -26,11 +33,13 @@
     :blur?     true
     :list-type :settings}])
 
+(defn navigate-back
+  []
+  (rf/dispatch [:navigate-back]))
+
```

Comment:
There seems to be many `navigate-back` functions popping up in the codebase, it would be nice to find a common place for navigation utils so we can share this function.

Reply:
I tried to push for that, But the response that I got is that it has a negligble effect.

Reply:
Well the main benefit is that if we want to rename the event `:navigate-back` to something like `:navigation.navigate-back` or `::navigation/navigate-back` then we can manage that in one place. Which imo would help avoid more bugs when refactoring stuff like that.

Reply:
Well the good news is that most of them are easy to find, so I'll create a chore issue soon for replacing these scattered functions with a single function 🙌

Reply:
Very much appreciated! Thanks!

Reply:
I've created the issue here: https://github.com/status-im/status-mobile/issues/19971 🍻